Remotion/Core/Logging.Log4Net/Log4NetLogger.cs

```diff
@@ -0,0 +1,134 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Globalization;
+using log4net.Core;
+using Remotion.Utilities;
+using MicrosoftLogger = Microsoft.Extensions.Logging.ILogger;
+using MicrosoftLoglevel = Microsoft.Extensions.Logging.LogLevel;
+using MicrosoftEventId = Microsoft.Extensions.Logging.EventId;
+using RemotionLoglevel = Remotion.Logging.LogLevel;
+
+namespace Remotion.Logging.Log4Net;
+
+/// <summary>
+/// Implements <see cref="Microsoft.Extensions.Logging.ILogger"/> to provide an implementation with log4net.
+/// </summary>
+public class Log4NetLogger : MicrosoftLogger
+{
+  public static Level Convert (MicrosoftLoglevel logLevel)
```

Comment:
Depending on the answer for whether we want the extension methods with Microsoft.Logging or not these might be made private. 

If we want the extension methods with the old loglevels, we should convert in the extension methods. If we want the microsoft loglevels only, we can make the converts private and remove the unused one.

Reply:
Done.